src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let robotAccountantsCreated = 0;
+
+  return number1 =>
+    number2 => {
+      robotAccountantsCreated++;
+
+      const makerErrorCondition = robotAccountantsCreated > 3
```

Comment:
```suggestion
      const isError = robotAccountantsCreated > 3
```
It would be great to name boolean variables according to this rule
![image](https://github.com/mate-academy/js_sum-with-closure/assets/96409563/0085a8e9-31a8-47c6-8291-be537711d044)